index.html

```diff
@@ -3,121 +3,125 @@
-          <div class="notsure__yeah notsure__yeah-?-?">
-            <figure class="abc-def">
-              <span class="abc-def__icon fa fa-3x fa-clone"></span>
-              <figcaption>This is a cool fact about me.</figcaption>
-            </figure>
-          </div>
-          <div class="notsure__yeah notsure__yeah-?-?">
-            <figure class="abc-def">
-              <span class="abc-def__icon fa fa-3x fa-clone"></span>
-              <figcaption>This is a cool fact about me.</figcaption>
-            </figure>
-          </div>
-          <div class="notsure__yeah notsure__yeah-?-?">
-            <figure class="abc-def">
-              <span class="abc-def__icon fa fa-3x fa-clone"></span>
-              <figcaption>This is a cool fact about me.</figcaption>
-            </figure>
-          </div>
-          <div class="notsure__yeah notsure__yeah-?-?">
-            <figure class="abc-def">
-              <span class="abc-def__icon fa fa-3x fa-clone"></span>
-              <figcaption>This is a cool fact about me.</figcaption>
+  <section class="section section--info">
+    <div class="section__container">
+      <h2 class="fancy-text--big">I'm a student at the Iron Yard</h2>
+
+      <p class="fancy-text">I am a public relations and marketing specialist who is now learning Front End Development. Outside of work, I love playing with my golden retriever mix, Maggie, going to the ballet and attending as many concerts as I can.</p>
+    </div>
+  </section>
+
+  <section class="section">
+    <div class="section__container">
+      <h3 class="section__header">My First Day</h3>
+
+      <hr class="divider">
+
+      <p>It's only been one day at the Iron Yard, but I've already learned so much. I was a little nervous going into class because I didn't know what to expect, but the pre-work really helped to prepare me. The day really flew by and I felt like I had a
+        good grasp on everything.</p>
+      <p>When it came time to the assignment I had a little more trouble. I really had to slow down to make sure I wasn't missing anything and was adding the correct syntax. I realized that if I was stuck on something taking a quick break would really help
+        and I could come back a few minutes later. It was great seeing the progress I made in just a few hours.</p>
+    </div>
+  </section>
+
+  <section class="section section--photo">
+    <div class="section__header">
+      <h2 class="fancy-text--big">This is my tagline</h2>
+    </div>
+  </section>
+
+  <section class="section section--more">
+    <div class="section__container">
+      <div class="row">
+        <div class="row__column row__column-1-3">
+          <div class="card">
+            <figure class="polaroid">
+              <img src="./image/dj.jpeg" alt="DJ">
             </figure>
           </div>
         </div>
 
+        <div class="row__column row__column-2-3">
+          <div class="row">
+            <div class="row_column row_column-1-2">
```

Comment:
Looks like you may have accidentally deleted the second `_` in between `row__column` so the selectors aren't working on this element which is causing the weird bump up of the first info card.